pilot/pkg/features/experimental.go

```diff
@@ -208,4 +217,41 @@ var (
 		"If enabled, ServiceEntries with wildcard hosts and dynamic dns resolution will be allowed for TLS traffic. "+
 			"This is a security risk, susceptible to SNI spoofing, and should be used with caution. "+
 			"Only consider using this feature if the client is trusted and you understand the risks.").Get()
+
+	PilotIgnoreResourcesEnv = env.Register(
+		"PILOT_IGNORE_RESOURCES",
+		"",
+		"If set, the resources set on this list will be ignored and never reconciled."+
+			"This value should be a comma-separated list of resources names."+
+			"Items on this list can be prefixed with a '*.' meaning a whole group should be ignored.",
+	).Get()
+
+	PilotIncludeResourcesEnv = env.Register(
+		"PILOT_INCLUDE_RESOURCES",
+		"",
+		"If set, and combined with 'PILOT_IGNORE_RESOURCES' the resources set on this list will not be ignored."+
+			"This value should be a comma-separated list of resources names."+
+			"Items on this list can be prefixed with a '*.' meaning a whole group should be included regardless of the ignore list.",
+	).Get()
+
+	// FetchResourceFilter is used to fetch the filters (ignore or include) from the
+	// flags.
+	FetchResourceFilter = func(filters string) []ResourceFilterConfig {
```

Comment:
we don't usually add code in the `features` package. maybe move this to `pkg/kube/kclient/crdwatcher.go`?